lib/fastlane/plugin/wpmreleasetoolkit/helper/deprecated.rb

```diff
@@ -0,0 +1,52 @@
+module Fastlane
+  module Helper
+    # A helper class to store deprecated methods and actions
+    class Deprecated
+      # Creates a project_root_folder Fastlane ConfigItem
+      #
+      # @return [FastlaneCore::ConfigItem] The Fastlane ConfigItem for the `PROJECT_ROOT_FOLDER` environment variable
+      #
+      def self.project_root_folder_config_item
+        verify_block = proc do
+          unless ENV['PROJECT_ROOT_FOLDER'].nil?
+            UI.deprecated('DEPRECATED: The PROJECT_ROOT_FOLDER environment variable and config item are deprecated and will be removed in a future version of the Release Toolkit. Please provide an explicit path for instead.')
+          end
+        end
+
+        FastlaneCore::ConfigItem.new(
+          key: :project_root_folder,
+          env_name: 'PROJECT_ROOT_FOLDER',
+          description: 'The path to the project root folder',
+          deprecated: true,
+          optional: true,
+          verify_block: verify_block,
+          type: String,
+          default_value: ENV['PROJECT_ROOT_FOLDER']
```

Comment:
Given the fact that this is picked up from `env_name`, I'm not sure we want it to be the default value like this?

WDYT?

Reply:
Good point, we really don't need a default value for the `project_root_folder` or `project_name` config items. Addressed in https://github.com/wordpress-mobile/release-toolkit/commit/7d0240c2b7fcaf6a276a139e1b6a7a9bac6af6e2